Core/Core/UIViews/EmptyView.swift

```diff
@@ -64,7 +64,7 @@ open class EmptyView: UIView {
         set { imageView?.image = newValue }
     }
 
-    public required init() {
+    public override init(frame: CGRect) {
```

Comment:
IBDesignablesAgent was crashing on storyboard views constantly, with the error:
"Fatal error: Use of unimplemented initializer 'init(frame:)' for class 'Core.EmptyView'", this is a fix for that